Created data preserve

NgachaIan opened the pull request:
In this PR I preserve the user data in the input fields after page reload

divinecharlotte reviewed (changes requested):
Hi @NgachaIan    , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Descriptive PR👍 
- Descriptive readme 👍 

 




 
![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)
## Required Changes ♻️

- [ ] kindly consider setting up Linters for this project by including the ` eslint` checks cause you should have five checks but currently you have 4 
please consider checking back this [link ](https://github.com/microverseinc/linters-config)and make sure you add the renters required for this project as mentioned in the link
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

osamaashraf6 reviewed (changes requested):
Hi @NgachaIan  ☺️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

![you-are-near](https://media.giphy.com/media/SSR17K6yZWvGCYrfUh/giphy.gif)

## Highlights ✔️

- [x] Good commit messages.
- [x] Descriptive READme file in a professional way.
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow

 ## Required Changes ♻️

- [ ] Kindly as stated by the previous reviewer 😃 kindly consider setting up Linters for this project by including the  eslint checks cause you should have five checks but currently, you have 4, please consider checking back this [link ](https://github.com/microverseinc/linters-config)and make sure you add the renters required for this project as mentioned in the link

![image](https://github.com/NgachaIan/Personal_Portfolio/assets/83360636/55d31254-8cfd-4401-afe6-85d15a3804d1)


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NgachaIan commented:
@osamaashraf6 Hello, I have added the .eslintrc.json file and corrected all the errors I had in the JavaScript file but I don't know why it's not running

frank1738 reviewed (approved):
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media4.giphy.com/media/tIeCLkB8geYtW/giphy.gif"/>

## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
